custom/ewsghana/reports/stock_levels_report.py

```diff
@@ -60,16 +60,17 @@ def headers(self):
 
     def get_prod_data(self):
         def get_months_until_stockout_icon(value):
+            stock_levels = CommtrackConfig.for_domain(self.config['domain']).stock_levels_config
             STOCKOUT = 0.0
-            LOW = 1
-            ADEQUATE = 3
+            LOW = stock_levels.understock_threshold
+            OVERSTOCK = stock_levels.overstock_threshold
             if float(value) == STOCKOUT:
                 return '%s <span class="icon-remove" style="color:red"/>' % value
             elif float(value) < LOW:
                 return '%s <span class="icon-warning-sign" style="color:orange"/>' % value
-            elif float(value) <= ADEQUATE:
+            elif LOW < float(value) < OVERSTOCK:
                 return '%s <span class="icon-ok" style="color:green"/>' % value
-            elif float(value) > ADEQUATE:
+            elif float(value) >= OVERSTOCK:
```

Comment:
not blocking, but I think should change - it seems a bit funny to use the all caps constant convention now that these are coming from the `stock_levels` object. can just replace this with `stock_levels.overstock_threshold`